QuadPaySDK/QuadPaySDKKeys.m

```diff
@@ -0,0 +1,62 @@
+//
+// Generated by CocoaPods-Keys
+// on 17/10/2022
+// For more information see https://github.com/orta/cocoapods-keys
+//
+
+#import <Foundation/NSDictionary.h>
+#import "QuadPaySDKKeys.h"
+
+@interface QuadPaySDKKeys ()
```

Comment:
These keys are set to the development values. They need to be recreated to the production values.